landscape_analysis.md

```diff
@@ -10,20 +10,28 @@ this proposal interact? (maximum of 250 words)
 
 <!-- 243/250 words -->
 
-Nibabel is the base layer for the Nipy (Neuroimaging in Python) community of projects.  It is the basic workbench and input/output layer on which higher-level libraries can build.
+Nibabel is the base layer for the Nipy (Neuroimaging in Python) community of projects.  It is the basic workbench and input/output (I/O) layer on which higher-level libraries can build.
```

Comment:
We don't use this abbreviation further down do we?

Reply:
I used it in my suggestions, that's why I added it - but we could just go with input/output bc it will count as just one anyways.